src/wasm-ast-parser.y

```diff
@@ -1158,7 +1158,7 @@ static void dup_text_list(WasmAllocator* allocator,
      * iterate through the string once. */
     const char* src = node->text.start + 1;
     const char* end = node->text.start + node->text.length - 1;
-    size_t size = end - src;
+    size_t size = (end > src) ? (end - src) : 0;
```

Comment:
You'll have to rebuild the prebuilt parser for this change; you can run `make update-bison` to do it.